syncd/Syncd.cpp

```diff
@@ -2791,49 +2791,95 @@ sai_status_t Syncd::processFlexCounterEvent(
     }
 
     auto groupName = key.substr(0, delimiter);
-    auto strVid = key.substr(delimiter + 1);
+    auto strVids = key.substr(delimiter + 1);
+    auto vidStringVector = swss::tokenize(strVids, ',');
```

Comment:
seems like this changes from single vid to multiple, will here be possible compatibility issue ? for example newer swss and older syncd that will expect 1 vid only and crash on deserialize when it will receive multiple ?

Reply:
Yes but it doesn't look like a valid use case.
Swss depends on sairedis, which means sairedis can be newer but won't be older than swss.
What do you think?